### ISSUE TITLE
Schedule an export dashboard and send via email

### ISSUE THREAD

**Issue by abeaulieu86** [sexibytes/sexigraf#403]
Hello, 

Do you have any way we can schedule a daily export of the dashboard VMware Multi Cluster Usage in PDF ? and send the result via email ? If not, do you have another solution (Ex: Scheduled snapshot of the dashboard and send the result by email) ?

Thanks

**Comment by rschitz** [comment]
Exporting dashboard is only available in the enterprise editin of grafana but with a simple powershell command you could get rendered png files that you could send by email in html format. Would that be enough?

**Comment by rschitz** [comment]
![sexigraf](https://github.com/user-attachments/assets/a5338bd3-dd17-4d4f-844f-0c3c0849f5d1)

**Comment by abeaulieu86** [comment]
Yes perfect for me. Do you have any instruction to do it ?

**Comment by rschitz** [comment]
yes, i'll make a blog post properly documented since you're not the first one to ask someting like this

**Comment by abeaulieu86** [comment]
Great!

Let me know when it's available please.

Thanks

Obtenir Outlook pour Android<https://aka.ms/AAb9ysg>
________________________________
